scripts/update_online.py

```diff
@@ -0,0 +1,27 @@
+import socket
+import os
+def arg(p=1):
+  try:
+    import sys
+    return sys.argv[p]
+  except:
+    return None
+os.chdir("..")
+
+lists = ["porn.txt","abuse.txt","ads.txt","basic.txt","crypto.txt","drugs.txt","everything.txt","facebook.txt","fraud.txt","gambling.txt","malware.txt","phishing.txt","piracy.txt","ransomware.txt","redirect.txt","scam.txt","smart-tv.txt","tiktok.txt","torrent.txt","tracking.txt","whatsapp.txt","youtube.txt"]
```

Comment:
Not a fan of this. Already doing this in the everything script. Having to update multiple places whenever we add a new list is not efficient.

Reply:
Do you have a suggestion? I am open to anything